docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -163,12 +161,76 @@ To deploy the connected registry module using the Azure CLI, save the following
 
 Use the information from the previous sections to update the relevant JSON values:
 
-- The environment variable `ACR_REGISTRY_CONNECTION_STRING` needs to be updated with the output from the `az acr connected-registry install renew-credentials` command above. You will need to manually add the `ParentGatewayEndpoint` with the IP or host address of the parent registry, `10.16.7.4` in this case. You will also need to select the proper protocol in `ParentEndpointProtocol`.
-- For each module in the manifest, you should update the registry endpoint to the IP address or hostname of the parent, `10.16.7.4` in this case.
+- The environment variable `ACR_REGISTRY_CONNECTION_STRING` needs to be updated with the output from the `az acr connected-registry install renew-credentials` command above. You will need to manually replace the `ParentGatewayEndpoint` with the $IOTEDGE_PARENTHOSTNAME. You will also need to select the proper protocol in `ParentEndpointProtocol`.
+- For each module in the manifest, you should update the registry endpoint to the $upstream.
 
 You will use the file path in the next section when you run the command to apply the configuration to your device.
 
-## Deploy the connected registry module on IoT Edge
+## Setup and deploy the connected registry module on nested IoT Edge
+This tutorial requires a nested Azure IoT Edge device to be set up upfront. You can use the [Deploy your first IoT Edge module to a virtual Linux device](https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux?view=iotedge-2020-11) quickstart guide to learn how to deploy a virtual IoT Edge device. To create a nested IoT Edge devices, follow the instructions [Tutorial: Create a hierarchy of IoT Edge devices](https://docs.microsoft.com/en-us/azure/iot-edge/tutorial-nested-iot-edge?view=iotedge-2020-11) to learn how to configure hierarchical IoT edge devices. The connected registry is deployed as a module on the nested IoT Edge device.
+
+Based on the tutorial, it overall includes the following steps:
+1. Create top level and lower level vms from existing template. The template will also install the iot agent on it. You can use the [Tutorial: Install or uninstall Azure IoT Edge for Linux](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge?view=iotedge-2020-11) to learn how to manually set up the machine if you need deploy from your own devices.
```

Comment:
Better to add a reminder here to let customer make sure they open the ports 8000, 443, 5671, 8883 on top level vm

Reply:
Resolved